db/migrate/20230117161324_add_user_id_to_orders.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserIdToOrders < ActiveRecord::Migration[7.0]
+  def change
+    add_column :orders, :user_id, :integer
+  end
```

Comment:
I think the better could be to add a reference instead of a column like 
`add_reference :orders, :user, foreign_key: true`